docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -0,0 +1,140 @@
+
+![Avalanche Faucet](intro-to-erc721s/1-faucet.png)
+
+## Creating the Smart Contract
+
+To create the smart contract, we are going to use [Open Zeppelin](https://docs.openzeppelin.com/). Open Zeppelin is a key tool for building smart contracts quickly and easily. While we are only scratching the surface in this tutorial, ample documentation is available on their website for you to read when you want to build more complex contracts.
+
+Open Zeppelin provides a [Contract Wizard](https://docs.openzeppelin.com/contracts/4.x/wizard) that will build out ERC contracts. To avoid any complex coding environments, we will use this to create our ERC721 contract.
+
+![Contract Wizard](intro-to-erc721s/2-contract-wizard.png)
+
+Select `ERC721` on the Contract Wizard to get started. This will create the contract in the [Solidity programming language](https://docs.soliditylang.org/en/v0.8.15/).
+
+As you can see, the template contract is bare-boned. We will fill out the information in the left panel to auto-populate it into our contract. Make sure you change to the ERC721 tab as you get started to make the proper contract.
+
+The Wizard auto-fills in a name and symbol for your NFT collection, which we will modify here. I am naming it `Photography` and giving it the symbol `FOTO`. If you chose your own files to use during this tutorial, you can choose a relevant name and symbol for your collection.
+
+The Base URI field listed here is the URL of the metadata folder uploaded to Pinata(for example, ours is `https://gateway.pinata.cloud/ipfs/QmYdWxbiwsfsYcW1CYQPgYujAc9FMLPG3fgFcxFskbSsFa`). Paste that into the Base URI field. After the Wizard adds our variables to the template, our contract should look like this:
+
+![Contract Wizard Populated](intro-to-erc721s/3-wizard-populated.png)
+
+Next, we will want to check the `Mintable` and `Auto Increment Ids` boxes. This will populate a mint function into our template that would handle the incrementation of token Ids on mint if we had more than one NFT in our collection. We still want it to auto-assign our 1 NFT, so we will check it.
+
+This automatically checks the `Ownable` button, which gives the safeMint function the `onlyOwner` modifier. This modifier indicates that only the owner of the smart contract will be able to successfully call the function.
+
+:::info
+**Note:** This modifier should be removed when creating a smart contract for a public mint. Otherwise, users would not be able to successfully mint the NFTs when calling the safeMint function. This tutorial only handles the owner’s wallet address, so it is being left in.
+:::
+
+Now, our contract is a little more populated:
+
+![Contract Wizard SafeMint](intro-to-erc721s/4-wizard-safemint.png)
+
+For this simple example, we will not add any additional functionality to the `safeMint` function. Currently, it mints one NFT to the address specified in the function call. There is no cost to mint the NFT other than the gas fee for the transaction itself. 
```

Comment:
Consider using contractions to make the text read more casually.

Ex: "We'll not add any..." instead of "We will not add any..."